### ISSUE TITLE
Milestone 2.20 issues and review

### ISSUE THREAD

**Issue by tiroj** [stipub/stixfonts#227]
As some may have noticed, I have been closing v2.20 and v2.20 tentative milestone issues as I address them in sources during the current round of work. This is mostly to help me whittle down the things needing our attention during the work, and there is probably some better way to manage this (maybe using labels). Obviously it isn’t ideal to close issues until the new version goes public and the issues are re-tested. So I will probably re-open a bunch of these issues when v2.20 goes live in the new year.

**Comment by Firestar-Reimu** [comment]
You said in [issue 222](https://github.com/stipub/stixfonts/issues/222):

> Complete review and revision of math –script kerning has been part of the v2.20 work nearing completion for release in the first part of 2022.

As 2022 has gone a half year, when will v2.20 release?

**Comment by tiroj** [comment]
@davidmjones ?

We delivered v2.20 to STI Pub for testing at the end of 2021. I am unsure what their schedule is or whether they want us to do any additional fixes before public release.

**Comment by Firestar-Reimu** [comment]
So can it be downloaded now? The source code is still v2.13

**Comment by tiroj** [comment]
That is correct. The font deliveries go through internal STI Pub testing before being made public. There were a _lot_ of changes in the 2.20 delivery, so I anticipate some feedback from that testing, and possibly additional work before public release.

**Comment by Firestar-Reimu** [comment]
Here is TexLive 2023 and STI Pub still have not finished testing the v2.20 fonts?

**Comment by Enivex** [comment]
Is there any particular reason why development versions of the font cannot be made public? Just don't release any official version until it's done.

You would get much more useful feedback if people knew which issues were actually fixed or not.

**Comment by tiroj** [comment]
I would be open to that, but it is STI Pub’s call.

**Comment by firai** [comment]
Are there particular individual(s) at STI Pub that we could contact to pitch this to? As mentioned above, it seems like STI Pub's money could go further and the delivery cycle could potentially be shortened if the community are allowed to help test the development versions, particularly since the community identified many of the issues in the first place.

**Comment by Firestar-Reimu** [comment]
I tried mailing to stix@aip.org but no one answers. Maybe we need @tiroj himself to push STI Pub to get on an [Agile development](https://learn.microsoft.com/en-us/devops/plan/what-is-agile-development)

**Comment by davidmjones** [comment]
> 

Thank you for your comments.  We are aware of the problems and will discuss them internally.

**Comment by Enivex** [comment]
Any news? It's now 2024

**Comment by mon-jai** [comment]
Any news?

**Comment by davidmjones** [comment]
> Any news?

I am working on getting caught up with this and develop a realistic schedule for completing this work, but the time I can devote to this project is very limited.  Thank you for your continued interest and patience.

**Comment by grivasgervilla** [comment]
And are the Sti Pub companied (ACS, AMS, IEEE...) truly committed with this project? I think that their review work for the last font version is in a stall point, right? I think that it's a pity this amazing work has such low priority on their agendas... This font family should evolve to be the main font used in every scientific text with even more weights, families (sans-serif or monospaced) added and more evolution to make the greatest scientific font out there, and for that money and prority are needed.

**Comment by kenmcd** [comment]
> but the time I can devote to this project is very limited.

Then let the font developer upload the current sources and beta fonts - and let people who do have the time and expertise get the review process going, and make some progress.
Just sitting on it is stupid.

There is a large ecosystem built around this font, as was the goal, and now many people depend on it.
Stalling development because one person "does not have the time" is ridiculous, and self-centered.

And do not come back with the old open source FU - blah, blah, you can fork it and do it yourself.
Upload the current v2.20 sources and we will be happy to do so.
Since you do not care enough to make the time, let others who do have the time, and do care enough, get it done.

**Comment by Enivex** [comment]
@kenmcd this is not helpful at all

**Comment by tiroj** [comment]
@kenmcd is a little heavy handed, but I sympathise with his frustration. Some additional information may be helpful: there are issues with some of the cut-in kerning data in the v2.20 fonts that should be addressed before they are put into use. Some of the super- and subscript positioning is ending up too tight, and needs to be adjusted. @davidmjones and I are having trouble aligning schedules to work on this and to gather other issues into work items, and I keep getting pulled onto other projects.

[As I recall, there is also at least one open question about Microsoft’s spec vs implementation of OT math layout, and I am unclear whether this might result in changes being needed in STIX Two, Cambria Math, and other fonts.]

**Comment by kenmcd** [comment]
Thanks for the info/update.
But I have confidence that your extensive knowledge and experience can more than handle all of this - if someone would just get out of the way.

LOL. _"This comment was marked as abuse."_ 
These days you give someone a kick-in-the-a$$ and it is "abuse."
Welcome to the World of Snowflakes.

**Comment by mon-jai** [comment]
@tiroj, is there any chance that the source of the proposed v2.20 and a nightly build could be released without approval from STI Pub?

That would be useful.

**Comment by tiroj** [comment]
Technically, that could be done as a fork within the context of the OFL license, but I wouldn’t be comfortable doing that without permission, since STI Pub hired me to do that work.

**Comment by grivasgervilla** [comment]
Hi, @tiroj.

Is there any information about the plans of STI Pub for the font? Could we know if STI Pub is taking care of this project?

Thank you very much for your amazing job.

**Comment by tiroj** [comment]
January 2025 update: I have started a new round of work on the STIX Two fonts. The priority is targeting issues that STI Pub reported in the 2.20 delivery, but I have also been through the issues logged since then and have flagged some for v2.25 (definitely or tentatively, as time and budget permit).
